linkinator.config.json

```diff
@@ -6,6 +6,7 @@
     "^http://localhost:9090",
     "^http://localhost:3000",
     "^http://localhost:15672",
-    "^http://192.168.99.100"
+    "^http://192.168.99.100",
+    "^http://stateless.co"
```

Comment:
Not sure I understand the reason to have this in here.

Reply:
There is no `https://stateless.co` and from CI http is blocked to outside. Has to blacklist that link.